proto/notification/service.proto

```diff
@@ -133,6 +135,7 @@ message ListSubscriptionsRequest {
   string search_keyword = 7;
   google.protobuf.BoolValue disabled = 8;
   string environment_id = 9;
+  string organization_id = 10;
```

Comment:
We must make this field required.

See for reference.
https://github.com/bucketeer-io/bucketeer/pull/1373/files#diff-78b5b99432b23b23b8badd9ac6427a3f50810aaa44064311f7822dd0f3779a82R263